app/views/dashboards/preferences.html.haml

```diff
@@ -24,12 +24,18 @@
     .col-md-12
       %h2.text-center=t("preferences.languages")
       .row
-        = form.simple_fields_for :skills, Project::LANGUAGES.map{|l| Skill.new(:language => l)} do |skill_form|
-          - language = skill_form.object.language
-          .col-xs-6.col-sm-4.col-md-2
-            %label.language-checkbox
-              = skill_form.check_box :language, {:id => language, :checked => current_user.skills.map(&:language).include?(language)}, language
-              = language
+        - Project::LANGUAGES.in_groups(2, '') do |main_col|
+          .col-xs-6.col-languages
```

Comment:
Rather than defining this super generic class which just sets `width` to 100% you can let Bootstrap take care of that by setting `.container-fluid` on it's parent. :)